unit_tests/src/test_process_stash.cpp

```diff
@@ -102,3 +103,10 @@ TEST_CASE("process_stash::erase_if")
     CHECK(irods::process_stash::find(h2));
     CHECK(irods::process_stash::handles().size() == 1);
 }
+
+// NOLINTNEXTLINE(readability-function-cognitive-complexity)
+TEST_CASE("trimPrefix does not go past end of string") {
+	char* testString{"OUTPUT"};
+	trimPrefix(testString);
+	REQUIRE(*testString == '\0');
+}
```

Comment:
If we deprecate the function, then we don't need a c++ unit test.

Reply:
Removed.